tools/syz-declextract/run.go

```diff
@@ -176,6 +205,9 @@ func worker(outputs chan output, files chan string, binary, compilationDatabase
 }
 
 func renameSyscall(syscall *ast.Call, rename map[string][]string) []*ast.Call {
+	if isAllowedDuplicate(syscall.CallName) {
```

Comment:
I don't understand relation to "duplicates" here. Why are these duplicates? Duplicates of what?
Perhaps this function needs a different name.